_includes/scripts.html

```diff
@@ -0,0 +1,132 @@
+<script src="js/d3.v3.min.js"></script>
+<script src="js/lodash.min.js"></script>
+<script src="js/app.js"></script>
+
+<script>
```

Comment:
Remove this `<script>` and add `<script>` tags in the markdown that define the net and call the "render" function

Reply:
Done.